src/components/BookReviewPage/BookReviewPage.jsx

```diff
@@ -0,0 +1,15 @@
+import SingleBookDetail from "../Booklisting/SingleBookDetail/SingleBookDetail"
+import "./BookReviewPage.css"
+
+function BookReviewPage({name, review, rating}){
+    <SingleBookDetail/>
```

Comment:
It's not doing anything with this tag, it can be removed. Don't forget to remove the imprt as well.